### PR TITLE
20_2 TreeList multiple selection demo: rewrite

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello](https://trello.com/c/PurkcTKk/1652-treelist-demos-selection-2-demos)
The text is based on https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/MultipleRecordSelectionModes/jQuery/Light/
https://github.com/DevExpress/devextreme-demos/pull/792

### CODE REVIEW

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 1]
```diff
@@ -1 +1,5 @@
...
\ No newline at end of file
+This demo sets **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode) to *"multiple"*. This allows users to select multiple rows using checkboxes or keyboard shortcuts.  Note that the default selection mode is non-recursive, that is, only the clicked row is selected. To allow users to select the dependent rows, set **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) to **true**. In this demo, you can use the control under the TreeList to toggle this mode.
+
+Click the checkbox in the leftmost column header to select all rows. If you do not need this default feature, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.
+
+The TreeList has an API to select multiple rows. For details, refer to the [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API) article.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]:
```suggestion
The TreeList includes an API to select multiple rows. For details, refer to the [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API) article.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 2]
```diff
@@ -1 +1,5 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo sets **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode) to *"multiple"*. This allows users to select multiple rows using checkboxes or keyboard shortcuts.  Note that the default selection mode is non-recursive, that is, only the clicked row is selected. To allow users to select the dependent rows, set **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) to **true**. In this demo, you can use the control under the TreeList to toggle this mode.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]:
```suggestion
This demo enables the *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to select multiple rows using checkboxes or keyboard shortcuts.

The checkbox in the leftmost column header selects all rows. If you want to disable it, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+
+Click the checkbox in the leftmost column header to select all rows. If you do not need this default feature, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]:
```suggestion
Multiple selection mode is non-recursive (when a row is selected, nested rows remain unselected). If users should select rows recursively, set the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) property to **true**. In this demo, you can use the checkbox under the TreeList to toggle this property.
```

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 4]
```diff
@@ -1 +1,7 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo enables the *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to select multiple rows using checkboxes or keyboard shortcuts.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: 1. Normally, we don't use articles with modes (just the way you did below).
2. I would have avoided "using" if possible.
```suggestion
This demo enables *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to press keyboard shortcuts or click checkboxes to select multiple rows.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 5]
```diff
@@ -1 +1,7 @@
...
+
+The checkbox in the leftmost column header selects all rows. If you want to disable it, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: >The checkbox in the leftmost column header selects all rows. If you want to disable **it**, set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false**.

I understand, "it" refers to "the checkbox" in your second sentence. However, this does not look quite clear.

```suggestion
The checkbox in the leftmost column header selects all rows. You can set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false** to disable the checkbox.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 6]
```diff
@@ -1 +1,7 @@
...
+
+Multiple selection mode is non-recursive (when a row is selected, nested rows remain unselected). If users should select rows recursively, set the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) property to **true**. In this demo, you can use the checkbox under the TreeList to toggle this property.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: TreeList is your component's name, you need no articles with it. If you want to preserve the definite article, add "component" to TreeList.

```suggestion
Multiple selection mode is non-recursive (when a row is selected, nested rows remain unselected). If users should select rows recursively, set the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) property to **true**. In this demo, you can use the checkbox under TreeList to toggle this property.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 7]
```diff
@@ -1 +1,7 @@
...
+
+The TreeList includes an API to select multiple rows. For details, refer to the [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API) article.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: Likewise, no need for an article here.

```suggestion
TreeList includes an API to select multiple rows. For details, refer to the [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API) article.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 8]
```diff
@@ -1 +1,7 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo enables *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to press keyboard shortcuts or click checkboxes to select multiple rows.
```
Comment (albertov05) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: `**selection**.[mode]`

Is the period in between the two words a typo? For example, do you mean:

`**selection** [mode]`

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 9]
```diff
@@ -1 +1,7 @@
...
+
+The checkbox in the leftmost column header selects all rows. You can set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false** and thus disable the checkbox.
```
Comment (albertov05) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]:
```suggestion
The checkbox in the leftmost column header selects all rows. You can set the [allowSelectAll](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#allowSelectAll) property to **false** to disable the checkbox.
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md [hunk 10]
```diff
@@ -1 +1,7 @@
...
+
+Multiple selection mode is non-recursive (when a row is selected, nested rows remain unselected). If users should select rows recursively, set the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#recursive) property to **true**. In this demo, you can use the checkbox under the TreeList component to toggle this property.
+
+TreeList includes an API to select multiple rows. For details, refer to the [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API) article.
```
Comment (albertov05) [inline JSDemos/Demos/TreeList/MultipleRowSelection/description.md]:
```suggestion
TreeList includes an API to select multiple rows. For details, refer to the following article: [Selection API](/Documentation/Guide/UI_Components/TreeList/Selection/#API).
```

File: JSDemos/Demos/TreeList/MultipleRowSelection/description.md (hunk 8 above)
Comment (pavel-2020) [reply JSDemos/Demos/TreeList/MultipleRowSelection/description.md]: The dot is there to denote this property: https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode.

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**